src/screens/UserManagement/UserRoleEntry.res

```diff
@@ -54,6 +74,11 @@ let make = () => {
     if usersData->Array.length === 0 {
       getUserData()->ignore
     }
+
+    if rolesAvailableData->Array.length == 0 {
+      getRolesAvailable()->ignore
```

Comment:
rather making a api call over here create a separate component for Role and make a api call over there